randobot/handler.py

```diff
@@ -556,30 +604,58 @@ async def ex_settings(self, args, message):
             if len(args) == 0:
                 await self.send_message(
                     'The following settings are available: '
-                    f"{' | '.join(draft.get('available_settings').get('minor').keys())}"
+                    f"{' | '.join(minor_pool.keys())}"
                 )
                 await self.send_message(
                     'Use !settings <setting> to view available values.'
                 )
             # List available values for specific setting
-            elif len(args) == 1 and args[0] in draft.get('available_settings').get('minor').keys():
-                setting = draft.get('available_settings').get('minor').get(args[0])
+            elif len(args) == 1 and args[0] in minor_pool.keys():
+                setting = minor_pool.get(args[0])
                 await self.send_message(
                     f'Available values for {args[0].capitalize()}: {", ".join(value for value in setting.keys())}'
                 )
         elif draft.get('status') == 'complete':
             if len(args) == 0:
-                setting = draft.get('drafted_settings').get('bans')
                 await self.send_message(
-                    f"Bans for this race: {', '.join(value.capitalize() for value in setting.keys())}"
+                    f"Bans for this race: {', '.join(value.capitalize() for value in bans.keys())}",
+                    pinned=True
```

Comment:
Not sure there's much point in pinning the bans? I doubt runners will need to refer back to them while racing.